src/server/v2.0/handler/usergroup.go

```diff
@@ -110,6 +110,9 @@ func (u *userGroupAPI) ListUserGroups(ctx context.Context, params operation.List
 	if err != nil {
 		return u.SendError(ctx, err)
 	}
+	if params.GroupName != nil && len(*params.GroupName) > 0 {
```

Comment:
https://github.com/goharbor/harbor/blob/main/src/server/v2.0/handler/replication.go#L187

Empty should an condition?